lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -890,7 +895,7 @@ def install_repo_configs(host, buildserver_url, package_name, build_version, cop
                                   repo_filename,
                                   copy_dir )
 
-          if host[:platform] =~ /cisco-5/
+          if host[:platform] =~ /cisco-nexus/
             to_path = "#{host.package_config_dir}/#{File.basename(repo)}"
```

Comment:
It's getting the correct file from our repo.  I am confident, because we have tested this.